src/core/cff_parser.js

```diff
@@ -885,6 +877,30 @@ var CFF = (function CFFClosure() {
 
     this.isCIDFont = false;
   }
+  CFF.prototype = {
+    duplicateFirstGlyph: function CFF_duplicateFirstGlyph() {
+      // Browsers will not display a glyph at position 0. Typically glyph 0 is
+      // notdef, but a number of fonts put a valid glyph so it must be
```

Comment:
Grammar nit: Shouldn't it perhaps be a "there" after "... valid glyph", i.e.
`// notdef, but a number of fonts put a valid glyph there so it must be`?